src/main/drivers/io.c

```diff
@@ -275,6 +275,17 @@ resourceOwner_e IOGetOwner(IO_t io)
     return ioRec->owner;
 }
 
+bool IOIsFree(IO_t io)
```

Comment:
Is 'IsFree' the right name for this? After all pins with `OWNER_SPI_PREINIT_IPU` or `OWNER_SPI_PREINIT_OPU` are owned.